http/oidc/src/main/java/org/wildfly/security/http/oidc/TokenValidator.java

```diff
@@ -263,7 +262,14 @@ public ErrorCodeValidator.Error validate(JwtContext jwtContext) throws Malformed
                 String atHash = jwtClaims.getStringClaimValue(AT_HASH);
                 String actualHash;
                 try {
-                    actualHash = getAccessTokenHash(accessTokenString, jwsAlgorithm);
```

Comment:
Hi @sugan0tech 
Thank you so much for the PR! 
I think instead of getting rid of the function, the Issue indicates moving the function. So, I think we can still keep the `getAccessTokenHash` function, but move it from the outer class, TokenValidator, to the inner class, TokenValidator#AtHashValidator. This way the code is cleaner and more easily understandable. 
Please let me know if you have any questions about it.

Reply:
@PrarthonaPaul  updated.

Reply:
Thanks!